src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+                ]
+              },
+              {
+                "issueId": "d056be64-829e-4a83-bfb1-e3232eb3a59b",
+                "priority": "CRITICAL",
+                "state": "CLOSED",
+                "title": [
+                  "Transaction query result is > 2.0 on 'Error fetching commitments (TF)'"
+                ]
+              },
+```
+
+The results can be further filtered by `entityGuid`. 
+
+**Request** 
+
+```graphql
+{
+  actor {
+    account(id: 1) {
+      aiIssues {
+        issues(filter: {priority: "CRITICAL", states: CLOSED, 
+          entityGuids: ["MXxBUE18QVBQTElDQVRJT058MzIxNjY2NA", "MXxBUE18QVBQTElDQVRJT058MzIxNzY1NA"]}) {
+          issues {
+            entityTypes
+            entityNames
+            entityGuids
+            issueId
+            priority
+            state
+            title
+          }
+          nextCursor
+        }
+      }
+    }
+  }
+}
+```
+
+**Results** 
+
+```json
+{
+  "data": {
+    "actor": {
+      "account": {
+        "aiIssues": {
+          "issues": {
+            "issues": [
+              {
+                "entityGuids": [
+                  "MXxBUE18QVBQTElDQVRJT058MzIxNjY2NA",
+                  "MXxBUE18QVBQTElDQVRJT058MzIxNzY1NA"
+                ],
+                "entityNames": [
+                  "hour-timeslice-writer (eu-production)",
+                  "metric-timeslice-service (eu-production)"
+                ],
+                "entityTypes": [
+                  "Application"
+                ],
+                "issueId": "1a3a0581-aefc-4027-91a0-a941072101c0",
+                "priority": "CRITICAL",
+                "state": "CLOSED",
+                "title": [
+                  "'Custom/CassandraClient/hour_timeslice_cluster_main/percentile/95th' > 0.5 for at least 5 minutes on 'hour-timeslice-writer (eu-production)'",
+                  "'Custom/CassandraClient/hour_timeslice_cluster_main/percentile/95th' > 0.5 for at least 5 minutes on 'metric-timeslice-service (eu-production)'"
+                ]
+              },
+              {
+                "entityGuids": [
+                  "MXxBUE18QVBQTElDQVRJT058MzIxNzY1NA"
+                ],
+                "entityNames": [
+                  "metric-timeslice-service (eu-production)"
+                ],
+                "entityTypes": [
+                  "Application"
+                ],
+                "issueId": "fd5fda9e-6bfe-469a-8cf7-822986401033",
+                "priority": "CRITICAL",
+                "state": "CLOSED",
+                "title": [
+                  "'Custom/CassandraClient/hour_timeslice_cluster_main/percentile/95th' > 0.5 for at least 5 minutes on 'metric-timeslice-service (eu-production)'"
+                ]
+              }
+            ],
+            "nextCursor": null
+          }
+```
+
+ </Collapser>
+  <Collapser
+     className="freq-link"
+     id="find"
+     title="Find issues by ID"
+   >
+
+The New Relic API lets you query by issue ID.
```

Comment:
there is no 'New Relic API'; should probably say 'NerdGraph also lets you...' or something